.pylintrc

```diff
@@ -304,7 +304,7 @@ valid-metaclass-classmethod-first-arg=mcs
 
 # List of member names, which should be excluded from the protected access
 # warning.
-exclude-protected=_asdict,_fields,_replace,_source,_make
```

Comment:
I would prefer not to reach into the `_thread` if at all possible. Are there other options?

Reply:
No, there are no other options.